devicon.json

```
@@ -2385,6 +2385,41 @@
             }
         ]
     },
+    {
+        "name": "detaspace",
+        "altnames": [
+            "Deta",
+            "Deta Space"
+        ],
+        "tags": [
+            "cloud",
+            "hosting",
+            "server"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "line",
+                "line-wordmark"
+            ],
+            "font": [
+                "line",
+                "line-wordmark"
+            ]
+        },
+        "color": "#ef39a8",
```

This color isn't used in any of the icons. Perhaps we could use the color of the outermost circle instead?
`#f73b95`
```suggestion
        "color": "#f73b95",
```